cfy_manager/scripts/sepolicy_deploy.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+echo "Setting up SELinux ${2} policy" | tee /tmp/sepolicy_deploy.log
+cd "${1}" || exit
+echo "Working in $( pwd )" | tee -a /tmp/sepolicy_deploy.log
+/usr/bin/make -f /usr/share/selinux/devel/Makefile "${2}.pp" 2>&1 | tee -a /tmp/sepolicy_deploy.log
```

Comment:
Thinking about this, we might be better generating the policies at build time and deploying them as part of the yum files?

Reply:
I would prefer to ship them in a human-readable version.

Reply:
The problem with that is we're then requiring build tools on production systems which it would be nicer not to require. It's already possible to query those policies using the selinux tools, so it'd probably be better to ship them compiled.